test/testdrive/kafka-commit.td

```diff
@@ -0,0 +1,40 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test that committing kafka offsets works
+
+$ kafka-create-topic topic=topic partitions=1
+
+$ kafka-ingest format=bytes topic=topic
+one
+two
+three
+
+> CREATE CONNECTION conn
+  FOR KAFKA BROKER '${testdrive.kafka-addr}'
+
+# Note that auto commit is not really safe to enable, but we have this here to
```

Comment:
Is this comment stale, now that we have the proper implementation in?

Reply:
good catch!